test/unit/NodeCompiler/CompileNodeToValueTest.php

```diff
@@ -173,15 +175,28 @@ public function testExceptionThrownWhenSpaceshipOperatorUsed()
         (new CompileNodeToValue())->__invoke(new Spaceship(new LNumber(5), new LNumber(3)), $this->getDummyContext());
     }
 
-    public function testExceptionThrownWhenConstUsed()
+    public function testExceptionThrownWhenUndefinedConstUsed()
     {
         $this->setExpectedException(
             UnableToCompileNode::class,
-            'Unable to compile constant expressions'
+            'Constant "FOO" has not been defined'
         );
         (new CompileNodeToValue())->__invoke(new ConstFetch(new Name('FOO')), $this->getDummyContext());
     }
 
+    public function testConstantValueCompiled()
+    {
+        define('BETTER_REFLECTION_TEST_CONST', 123);
```

Comment:
This needs to pass through a `defined(...) || define(...)` to avoid issues if the test is executed twice. Consider using `uniqid()` instead?